VirtoCommerce.CustomerModule.Web/Scripts/blades/member-detail.js

```diff
@@ -110,6 +110,7 @@
     $scope.fetchOrganizations = function ($select) {
         $select.page = 0;
         $scope.organizations = [];
+        getCustomerOrganizartions();
```

Comment:
Typo in the method name.
 And I would like use another name for this method some like this `loadCustomerOrganization` because this method doesn't return anything.

Reply:
Duplicates organizations - tested.
Typo fixed.